### PR TITLE
feat[WIP]:Tasks

### PR DESCRIPTION
**Pull request by saadahmadhere**
In this feature:
1. Users can add and delete tasks.
2. User is prompted to a modal when she clicks on the "add to task" button.
3. Basic form validation (default ones) is implemented.

### CODE REVIEW

File: src/Components/Modal/Modal.jsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+              }
+              required
+            />
+          </label>
+          <div className="action_buttons ">
+            <button
+              className="btn btn_outline px_3 py_1"
+              onClick={(e) => {
+                e.preventDefault();
+                return setShowModal(false);
```
Comment (AnuragSahu11) [inline src/Components/Modal/Modal.jsx:77]: do you need to use return, just using setShowModal will do the work I think

**Review by AnuragSahu11 — APPROVED** [review]
Nice Ui, good structured code , added a suggestion, other than that ready to merge

**Review by krate27 — APPROVED** [review]
Things look good to merge.

Good use of variables and it seems good UI to experience.

**Review by prachi-sahani — COMMENTED** [review]
Overall the code is well structured and good to merge

**Suggestion:**
- use a little smaller fonts in tasks page
- better naming convention can be used 
   -  task handler can be named as addTaskHandler, something which signifies the action being taken
   - instead of naming the component as Modal.jsx you can use some meaningful name(in case you plan to use modal at some other place in the application)

File: src/Components/Modal/Modal.jsx (hunk 1 above)
Comment (saadahmadhere) [reply src/Components/Modal/Modal.jsx:77]: Roger that. Fixed in https://github.com/saadahmadhere/Stopphur/pull/5/commits/21ce25818b20efcb604b39ec27e98dfa068c4a79

**Comment by saadahmadhere** [comment]
> Overall the code is well structured and good to merge
> 
> **Suggestion:**
> 
> * use a little smaller fonts in tasks page
> * better naming convention can be used
>   
>   * task handler can be named as addTaskHandler, something which signifies the action being taken
>   * instead of naming the component as Modal.jsx you can use some meaningful name(in case you plan to use modal at some other place in the application)

Thank you for the in-depth review. Will take care of the pointers given.

**Pull request merged**